### ISSUE TITLE
Choose which files merge in Send Only mode

### ISSUE THREAD

**Issue by vk496** [syncthing/syncthing#4478]
Scenario: I have enabled "Send Only" mode for a repository and is out of date (new files).

Request: In the overview of changes, the posibility of select or deselect which one do yo want to apply to your repository (preview mode?) and a button to apply that modification permanently.

Salu2

**Comment by calmh** [comment]
This seems like it could be useful, but I would not hold my breath.

**Comment by nrambaud** [comment]
Currently, the mode is settable at folder level. It could be interesting to select mode at device level in "shares" tabs just near it's possible to select the encryption code.

**Comment by agung2001** [comment]
I agree with @nrambaud if we have a sync mode on a remote device level it could be helpful for a couple of reasons :
- It will eradicate a problem for clients if they set the wrong folder path on their local computer, we can set "Send Only" mode and then "Send & Receive" after we make sure they set the folder path correctly.
- We can use this feature to share the final project with a client without worrying about anything regarding changes that are made to the folder.

**Comment by acolomb** [comment]
If you're trying to share folders to different devices in different modes, you can simply add the folder twice locally, with different settings. As long as the remote devices they're shared with don't overlap, it should be safe.
